### ISSUE TITLE
VPK v2 Support?

### ISSUE THREAD

**Issue by 0xFurtiv** [ValvePython/vpk#25]
On [developer.valvesoftware.com](https://developer.valvesoftware.com/wiki/VPK_File_Format#VPK_readers), this project is listed as supporting VPK v1 and v2. Is there support for writing VPK v2 archives? If not, is there a plan to support it?

**Comment by rossengeorgiev** [comment]
Yes, you can open v1 and v2. The creation of VPK files was only added later and is only for v1, single file.

Ideally, it would be best to rewrite the library and combine the reading and writing into a single class, but I have no plans to do that any time soon

**Comment by rossengeorgiev** [comment]
`v1.4.0` now creates VPK v2 files by default. Chunking is still not supported, and would be best achieved by a full rewrite

**Comment by juliogaray** [comment]
Dear Rossen,
Would it be possible to add an option parameter to the CLI app to chose whether to export to v1 or v2 (e.g. -v1 or -v2)? I actually need to create v1 VPKs, but I don't see a way to do it with the current version. Thanks for your consideration!

**Comment by rossengeorgiev** [comment]
@jgg1971 there already is `-cv`, see https://github.com/ValvePython/vpk/blob/3ff641e5fa7cf53a12c886153db0aab7a7872821/vpk/cli.py#L32

**Comment by juliogaray** [comment]
OK, sorry, I didn't look at the source code. This option is not shown in the readme, so I didn't think it was implemented. Thanks for the quick answer and good work!
